scripts/checks/verify_supply_chain.py

```
@@ -156,6 +157,12 @@ def verify_workflow_coverage() -> list[str]:
         missing.append(
             "build workflow should not rely on macos-latest for architecture coverage"
         )
+    scorecard = read_workflow(
+        Path(".github/workflows/ossf-scorecard.yml"), "ossf scorecard", missing
+    )
+    for token in ["develop", "main", "push", "schedule", "ossf-scorecard"]:
+        if scorecard and token not in scorecard:
+            missing.append(f"ossf scorecard workflow missing token: {token}")
```

Ruff (0.15.6)

[warning] Use `list.extend` to create a transformed list

(PERF401)